src/hipscat/io/paths.py

```diff
@@ -85,6 +86,20 @@ def get_healpix_from_path(path: str) -> HealpixPixel:
     return HealpixPixel(int(order), int(pixel))
 
 
+def pixel_catalog_files(
+        catalog_base_dir: FilePointer,
+        pixels: List[HealpixPixel],
+        storage_options: Dict | None = None
+):
+    fs, _ = get_fs(catalog_base_dir, storage_options)
+    base_path_stripped = catalog_base_dir.removesuffix(fs.sep)
+    return [fs.sep.join([base_path_stripped,
+                         f"{ORDER_DIRECTORY_PREFIX}={pixel.order}",
+                         f"{DIR_DIRECTORY_PREFIX}={pixel.pixel // 10000 * 10000}",
```

Comment:
```suggestion
                         f"{DIR_DIRECTORY_PREFIX}={pixel.dir}",
```

Reply:
Wooah I didn't know we had this that's cool